modules/ROOT/pages/security.adoc

```diff
@@ -1,6 +1,12 @@
-= Supported Authentication Methods
+:page-aliases: supported-authentication-methods.adoc
```

Comment:
This can't be the first line of the file. 
page-aliases must go below the title. 

```
= Anypoint DataGraph Security
:page-aliases: supported-authentication-methods.adoc
```